### PR TITLE
Remove UMA properties from identity.xml files

### PR DESCRIPTION
**Pull request by hwupathum**

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 1]
```diff
@@ -415,13 +415,6 @@
                 <IsRefreshTokenAllowed>{{oauth.grant_type.jwt_bearer.allow_refresh_tokens}}</IsRefreshTokenAllowed>
             </SupportedGrantType>
             {% endif %}
-            {% if oauth.grant_type.uma_ticket.enable is sameas true %}
-            <SupportedGrantType>
-                <GrantTypeName>urn:ietf:params:oauth:grant-type:uma-ticket</GrantTypeName>
-                <GrantTypeHandlerImplClass>{{oauth.grant_type.uma_ticket.grant_handler}}</GrantTypeHandlerImplClass>
-                <GrantTypeValidatorImplClass>{{oauth.grant_type.uma_ticket.grant_validator}}</GrantTypeValidatorImplClass>
-            </SupportedGrantType>
-            {% endif %}
```
Comment (senthalan) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:424]: how someone using this as a connector can configure these ?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -1605,11 +1598,6 @@
                        enable="{{event.default_listener.client_certificate_authentication_handler.enable}}"/>
 
         <!-- Introspection Data Providers -->
-        <EventListener id="uma_introspection_data_provider" type="org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"
-                name="org.wso2.carbon.identity.oauth.uma.permission.service.impl.UMAIntrospectionDataProvider"
-                orderId="{{event.default_listener.uma_introspection_data_provider.priority}}"
-                enable="{{event.default_listener.uma_introspection_data_provider.enable}}">
-        </EventListener>
```
Comment (senthalan) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:1612]: how someone using this as a connector can configure these ?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 1 above)
Comment (hwupathum) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:424]: These PRs for IPK therefore, will not be used as a connector

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 2 above)
Comment (hwupathum) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:1612]: These PRs for IPK therefore, will not be used as a connector

**Review by senthalan — APPROVED** [review]

**Pull request merged**